myapp/routes/users.js

```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+var router = express.Router();
+
+/* GET users listing. */
+router.get('/', function(req, res, next) {
```

Comment:
What is need of this route?
Always add only required code, do not add any extra and duplicate code.
Add code that fulfill the given requirements